code-prettify.php

```diff
@@ -46,7 +46,7 @@ function add_prettify_scripts() {
 	wp_localize_script(
 		'code-prettify',
 		'codePrettifyLoaderBaseUrl',
-		plugins_url( 'prettify', __FILE__ )
+		[ plugins_url( 'prettify', __FILE__ ) ]
```

Comment:
This would make the JS look like `var codePrettifyLoaderBaseUrl = [ 'https://example.com...' ]` which will break the JS part using this variable here:

https://github.com/kasparsd/code-prettify/blob/96e693b4cc0d043d4626abbb97c721613f0dcf69/prettify/run_prettify.js#L33

Maybe we should update this to use `wp_add_inline_script()` like so:

```php
wp_add_inline_script( 
    'code-prettify', 
    sprintf( 
        'var codePrettifyLoaderBaseUrl = %s;', 
        wp_json_encode( plugins_url( 'prettify', __FILE__ ) ) 
    ), 
    'before'
);
```